File: server/src/main/java/com/tp/holdem/server/GameParams.java
package com.tp.holdem.server;

import lombok.Builder;
import lombok.Value;

@Builder
@Value
class GameParams {
	private final int playerCount;
	private final int startingChips;
	private final int bigBlindAmount;
	private final int smallBlindAmount;
	private final int port;
}

File: server/src/main/kotlin/com/tp/holdem/server/GameParams.kt
package com.tp.holdem.server

internal data class GameParams(
        val playerCount: Int = 0,
        val startingChips: Int = 0,
        val bigBlindAmount: Int = 0,
        val smallBlindAmount: Int = 0,
        val port: Int = 54555
)
